### PR TITLE
Add support for sfence.vma instruction

### PR DESCRIPTION
**Pull request by buildandcrash**
- [X] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [ ] I wrote some lines in the [book](https://github.com/radareorg/radare2book) (optional)

**Description**

This pull request adds support for the sfence.vma instruction to the RISC-V plugin in Radare2 and fixes (#23021).

**Changes Made:**

Introduced sfence.vma instruction mnemonic, operand definition, and opcode matching in riscv.c.
Updated the opcode table (riscv_builtin_opcodes) to include sfence.vma with its corresponding opcode values and operand specifications.
Reason for Change:
The sfence.vma instruction is part of the RISC-V ISA and is used for flushing virtual memory address translations. Including support for sfence.vma enhances the completeness of the RISC-V plugin in Radare2, enabling better analysis and disassembly capabilities for binaries that utilize this instruction.

**Testing:**

Manually tested the functionality by disassembling binaries containing sfence.vma instructions.
Verified proper decoding and disassembly output alignment with RISC-V specifications.

Before:

```
rasm2 -D -a riscv -b 64 73000012
0x00000000   4                 73000012  invalid
```

After:

```
rasm2 -D -a riscv -b 64 73000012
0x00000000   4                 73000012  sfence.vma zero, zero
```

**Additional Notes:**

This change ensures compatibility with the latest RISC-V ISA extensions and improves the utility of Radare2 for analyzing modern RISC-V binaries.

### CODE REVIEW

**Comment by trufae** [comment]
Sweet! thanks for your contribution! would you like to continue extending the stock disassembler to support more VMA instructions? I bet there will be other pretty common ones around the same code.

Also, how is this instruction recognized by the analizer? because it should be reported as a different family (privileged or thready maybe)

File: libr/arch/p/riscv/riscv-opc.c [hunk 1]
```diff
@@ -660,6 +660,7 @@ static const struct riscv_opcode riscv_builtin_opcodes[] = {
 { "hret",      "I",   "",     MATCH_HRET, MASK_HRET, match_opcode, 0 },
 { "mret",      "I",   "",     MATCH_MRET, MASK_MRET, match_opcode, 0 },
 { "dret",      "I",   "",     MATCH_DRET, MASK_DRET, match_opcode, 0 },
+{ "sfence.vma","I", "s,t", MATCH_SFENCE_VMA, MASK_SFENCE_VMA, match_opcode, 0 },
```
Comment (trufae) [inline libr/arch/p/riscv/riscv-opc.c:663]: align columns to match the other rows

**Comment by buildandcrash** [comment]
Thank you! Contributing to radare2 would be an honor. I will add support to anything missing as I go, since I am working on RISC-V projects these days.

Also, the `sfence.vma` instruction was a replacement for `sfence.vm` which is considered an "improved" privileged instruction

**Comment by trufae** [comment]
Awesome! glad to have some riscv improvements, i love this arch but i can't spent fulltime on it yet. Btw if you care about decompilation maybe you can also check this PR https://github.com/wargio/r2dec-js/pull/291

**Pull request merged**